antismash/common/record_processing.py

```diff
@@ -46,44 +45,34 @@ def parse_input_sequence(filename: str, taxon: str = "bacteria", minimum_length:
         raise TypeError("minimum_length must be an int")
 
     records = []  # type: List[SeqRecord]
-    if not os.path.exists(filename):
-        msg = "Sequence file not found: %r" % filename
-        logging.error(msg)
-        raise ValueError(msg)
-
     try:
         record_list = list(seqio.parse(filename))
-        if not record_list:
-            raise AntismashInputError('no records could be read from file %r' % filename)
-        for record in record_list:
-            if isinstance(record.seq.alphabet, Bio.Alphabet.ProteinAlphabet):
-                raise AntismashInputError("protein records are not supported")
-            if minimum_length < 1 \
-                    or len(record.seq) >= minimum_length \
-                    or 'contig' in record.annotations \
-                    or 'wgs_scafld' in record.annotations \
-                    or 'wgs' in record.annotations:
-                records.append(record)
-    except (ValueError, AssertionError, PermissionError) as err:
-        logging.error('Parsing %r failed: %s', filename, err)
-        raise AntismashInputError(str(err)) from err
-    except AntismashInputError:
-        raise
     except Exception as err:
-        logging.error('Parsing %r failed with unhandled exception: %s',
-                      filename, err)
+        logging.error('Parsing %r failed: %s', filename, err)
         raise AntismashInputError(str(err)) from err
 
+    for record in record_list:
+        if minimum_length < 1 \
+                or len(record.seq) >= minimum_length \
+                or 'contig' in record.annotations \
```

Comment:
Does this make sense? We can't handle contig/WGS_SCAFLD/WGS records for now, and on the web service the downloader takes care of these.

Reply:
The condition is unchanged from what it was previously, it will at a later stage (in `pre_process_sequences()`) raise an error on those cases. I'm happy enough to move that error to the parse step instead, if you'd prefer.

Reply:
Nah, we probably just need to revisit this if/when we add a `ncbi-acc-download` dependency to deal with WGS records.